tools/vf-component-library/src/site/patterns/boilerplate-generic-embl-ebi.njk

```diff
@@ -0,0 +1,129 @@
+---
+title: Boilerplate for a generic EMBL-EBI page
+subtitle: Use this boilerplate pattern for generic content-focused pages at EMBL-EBI.
+section: patterns
+date: 2018-08-22 12:24:50
+tags: posts
+layout: layouts/boilerplate.njk
+---
+<!DOCTYPE html>
+<html lang="en" class="vf-no-js">
+<head>
+  <!-- See the VF No JS docs: https://stable.visual-framework.dev/components/vf-no-js/ -->
+  {% render "@vf-no-js" %}
+  <title>{{ pagination.items[0].name or title or (renderData and renderData.title)}} | {{ siteConfig.siteInformation.title }}</title>
+
+  <meta charset="utf-8">
+  <meta content="width=device-width, initial-scale=1, minimum-scale=1" name="viewport">
+
+  <!-- See the VF Favicon docs: https://stable.visual-framework.dev/components/vf-favicon/ -->
+  {% render "@vf-favicon" %}
+  <!-- See the embl-content-meta-properties docs: https://stable.visual-framework.dev/components/embl-content-meta-properties/ -->
+  {% render "@embl-content-meta-properties", {"meta_who": "Group Jane", "meta_where": "EMBL-EBI", "meta_what": "research", "meta_active": "what"} %}
+
+  {% if serverInfo.environment == "development" -%}
+  <link rel="stylesheet" media="all" href="{{ '/css/styles.css' | url }}" />
+  {%- else -%}
+  <link rel="stylesheet" href="https://{{ siteConfig.vfVersionPrefix }}assets.emblstatic.net/vf/{{ siteConfig.vfVersion }}/css/styles.css">
+  {%- endif %}
+  <!--
+    Want to build a site with the Visual Framework?
+    ---
+    You can copy the CSS and JS you find here, but for the best experience see the building guide:
+    https://stable.visual-framework.dev/building/
+  -->
+</head>
+<body class="vf-body vf-stack">
+
+<style>
+.ebi-header-footer.masthead-black-bar {
```

Comment:
is this because the gdpr banner loads first? Ideally, we'd want this in the actual CSS somehow - I did make a start before Xmas on something like this for ordered spacing depending on the preceding component -- but not sure where the CSS should live

Reply:
Whoops, added that as a fix elsewhere didn't mean to commit it!

Reply:
Now removed.